integration-tests/support/pages/tabs/PipelinerunsTabPage.ts

```diff
@@ -65,11 +65,12 @@ export class TaskRunsTab {
       { name: `${pipelineName}-clair-scan`, task: 'clair-scan', status: 'Succeeded|Test Failures' }, // Adding Test Warnings as some packages might have medium vulnerabilities sometimes
       { name: `${pipelineName}-clamav-scan`, task: 'clamav-scan', status: 'Succeeded' },
       { name: `${pipelineName}-label-check`, task: 'label-check', status: 'Succeeded' },
-      {
-        name: `${pipelineName}-optional-label-check`,
-        task: 'label-check',
-        status: 'Succeeded|Test Failures',
-      }, // Adding Test Failures as This optional check currently gives warning.
+      { name: `${pipelineName}-show-sbom`, task: 'show-sbom', status: 'Succeeded' },
+      // {
+      //   name: `${pipelineName}-optional-label-check`,
+      //   task: 'label-check',
+      //   status: 'Succeeded|Test Failures',
+      // }, // Adding Test Failures as This optional check currently gives warning.
```

Comment:
Do we want thins code commented on? Is there a plan to uncomment soon?

Reply:
😕 Its skipped, not sure in future they enable it back so thought of just commenting it.

Reply:
wdyt?

Reply:
If it is not sure, I would remove it. We can find it in git history if we need it later :slightly_smiling_face:

Reply:
removed 🙂